tests/ha/multi_pod_failure/test_multi_data_pod_failure.py

```diff
@@ -645,103 +614,174 @@ def test_server_data_kpods_fail_during_ios(self):
         LOGGER.info("STARTED: Test to verify continuous IOs while k server and data pods are "
                     "failing one by one")
 
-        LOGGER.info("Step 1: Perform Continuous WRITEs-READs-verify during k server and data pods "
-                    "are going down")
+        event = threading.Event()  # Event to be used to send when pods going down
```

Comment:
for both of these tests we would need skip marker as we are deleting while pods going down. skip marker can be "Functionality not supported with DTM0"